_posts/2017-10-20-jbhc17-from-my-perspective.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "#jbhc17 from my perspective"
+date: 2017-10-20 14:03:00 +0200
+categories: meetup
+author_name: Jacek Smolak
+author_url : /author/jaceksmolak
+author_avatar: jaceksmolak
+show_avatar : true
+read_time : 4
+show_related_posts: false
+square_related:
+feature_image: posts/2017-10-22-jbhc17-from-my-perspective/poster.jpg
+---
+
+When I first heard about [JB][jbrains] coming to HolidayCheck to work with us I was excited.
```

Comment:
You state that you were excited but in the next sentence you admit that at the time you didn't know who JB was, why the excitement?

Reply:
I was wondering as well, but then he is writing that he watched some talks of him, so I guess he meant  "didn't know very well" or "didn't know in person"

Reply:
Yes. I was excited about him visiting us, but also I did not know him that much (mostly if not only from the 7:26 talk). Like Sandro Mancuso - I know him from Software Craftsman and some tweets. That's all. And if he were to come to HC I would be like super excited :)